tstyche.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "testFileMatch": [
+    "**/*.test-d.ts"
```

Comment:
this could be limited to the `test` directory

Reply:
Right. This should slightly improve the performance.